frontend/tsconfig.json

```diff
@@ -17,10 +17,10 @@
     },
     "target": "es2015",
     "typeRoots": ["node_modules/@types"],
-    "lib": ["es2018", "dom"]
+    "lib": ["es2018", "dom"],
+    "esModuleInterop": true
```

Comment:
Is there any kind of issue that "esModuleInterop" flag solves for us?

Reply:
I thought that it might solve the initial issue, but that wasn't the case. Anyway, this flag is highly advised to be turned to true, I've encountered already issues that were solved by turning it on.
> We highly recommend applying it both to new and existing projects
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html